addons/intel/e810.go

```diff
@@ -76,12 +76,33 @@ func getDefaultUblxCmds() []E810UblxCmds {
 		ReportOutput: false,
 		Args:         []string{"-p", "CFG-MSG,0xf0,0x03,0"},
 	}
+	// Ublx command to disable VTG messages
+	cfgMsgDisableVTG := E810UblxCmds{
```

Comment:
why not make this changes in PTPconfig ? there is more flexibility  for changes that way than putting them in code ?

Reply:
I think this is not something users would like to tweak with. As opposed to NAV-TIMELS message, that users of other constellations than GPS might want to disable, these messages do nothing for T-GM but harm.